chaoscenter/graphql/server/pkg/chaoshub/handler/handler_test.go

```diff
@@ -368,8 +368,8 @@ func TestGetExperimentData(t *testing.T) {
 // TestListPredefinedWorkflowDetails is used to test the ListPredefinedWorkflowDetails function
 func TestListPredefinedWorkflowDetails(t *testing.T) {
 	// given
-	succeedProjectID := uuid.New().String()
-	succeedName := uuid.New().String()
+	succeedProjectID := "project-1"
+	succeedName := "workflow-hub-1"
```

Comment:
same

Reply:
We can kindoff refine the names and stuff, but we will require a defined file system to pass the tests.